.github/workflows/deliver.yml

```
@@ -79,21 +66,21 @@ jobs:
 
       - name: Get Package name
         run: |
-          echo "PACKAGE_NAME=$(sed -nE 's/.*package="(.*)".*/\1/p' package/app/android/app/src/main/AndroidManifest.xml)" >> $GITHUB_ENV
+          echo "PACKAGE_NAME=$(sed -nE 's/.*package="(.*)".*/\1/p' packages/app/android/app/src/main/AndroidManifest.xml)" >> $GITHUB_ENV
 
       - name: Create google service account file
         env:
           GOOGLE_SERVICE_ACCOUNT_KEY_JSON_BASE64: ${{ secrets.GOOGLE_SERVICE_ACCOUNT_KEY_JSON_BASE64 }}
-        working-directory: package/app
+        working-directory: packages/app
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:6: Double quote to prevent globbing and word splitting

(shellcheck)